packages/cinterop/src/jvm/kotlin/io/realm/internal/interop/RealmInterop.kt

```diff
@@ -38,7 +42,16 @@ actual object RealmInterop {
     // TODO API-CLEANUP Maybe pull library loading into separate method
     //  https://github.com/realm/realm-kotlin/issues/56
     init {
-        System.loadLibrary("realmc")
+        if (isAndroid()) {
```

Comment:
Should we consider abstracting the platform specific paths into some platform specific `initInterop`/`scheduler`  instead?